match/Dockerfile

```diff
@@ -1,16 +1,13 @@
 FROM golang:1.13.7-alpine
 
-WORKDIR /go/src/github.com/TempleEight/spec-golang/match/
+WORKDIR /match
+
+COPY go.mod go.sum ./
+RUN go mod download
 
 COPY . .
 COPY config.json /etc/match-service/
 
-RUN apk add --no-cache git
-
-RUN go get github.com/gorilla/mux
-RUN go get github.com/lib/pq
-RUN go get github.com/asaskevich/govalidator
-
 RUN go build -o match
 
 ENTRYPOINT ./match
```

Comment:
Can we get a new-line at the end of this file to keep in line with the other Dockerfile?